src/facade/dragonfly_connection.cc

```diff
@@ -176,28 +180,33 @@ void LogTraffic(uint32_t id, bool has_more, absl::Span<RespExpr> resp) {
   char stack_buf[1024];
   char* next = stack_buf;
 
-  // We write id, timestamp, has_more, num_parts, part_len, part_len, part_len, ...
+  // We write id, timestamp, db_index, has_more, num_parts, part_len, part_len, part_len, ...
```

Comment:
have you seen my comment about emitting "SELECT db_index" once when the logger starts instead of writing it for every record?

Reply:
Yes, I did, we save 4 bytes per entry... Is it worth it? 😮